pkg/provider/aws/resources/lambda_test.go

```diff
@@ -0,0 +1,77 @@
+package resources
+
+import (
+	"testing"
+
+	"github.com/klothoplatform/klotho/pkg/annotation"
+	"github.com/klothoplatform/klotho/pkg/core"
+	"github.com/klothoplatform/klotho/pkg/core/coretesting"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_LambdaCreate(t *testing.T) {
+	eu := &core.ExecutionUnit{AnnotationKey: core.AnnotationKey{ID: "test", Capability: annotation.ExecutionUnitCapability}, DockerfilePath: "path"}
+	cases := []struct {
+		name    string
+		lambda  *LambdaFunction
+		vpc     bool
```

Comment:
This appears to be unused in the test. I know we'll probably have this eventually, but the case struct should match with what is actually tested (so we don't forget to implement it).

Reply:
oh the vpc is from leftover when create wasnt just responsible for the name, good catch i will remove